src/Controllers/WorkItemController.php

```
@@ -80,7 +82,20 @@ public function index(): void
             return;
         }
 
-        $workItems = HLWorkItem::findListByProjectIdForOrg($this->db, $projectId, $orgId);
+        $reorderMode = $this->request->get('reorder') === '1';
+        $allWorkItems = HLWorkItem::findListByProjectIdForOrg($this->db, $projectId, $orgId);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\StratFlow\Models\HLWorkItem' in method 'index'. (undefined)

(StaticAccess)